### PR TITLE
f-strings implemented and pep8 styling

### PR DESCRIPTION
**Pull request by mehta-raghav**

### CODE REVIEW

File: rock_paper_scissor.py [hunk 1]
```diff
@@ -51,17 +52,18 @@ def round_win(player1, player2, player1_score, player2_score):
...
     if player1_score > player2_score:
-        print("---------------")
+        print("#"*128)
         return "!!! PLAYER 1 WON THE GAME !!!"
     elif player1_score < player2_score:
-        print("---------------")
+        print("#"*128)
         return "!!! COMPUTER WON THE GAME !!!"
     else:
         print("---------------")
```
Comment (kdipippo) [inline rock_paper_scissor.py]: You missed a line here. These can all be placed to the top of the function as well, like so:
```py
def winner(player1_score, player2_score):
   print("#"*128)
```
Then it doesn't have to be repeated 3 times for each conditional.

Comment (kdipippo) [reply rock_paper_scissor.py]: I also think 128 is too excessive for the number of pound signs shown. The original snippet had 15 hyphens.

**Comment by kdipippo** [comment]
Running pep8 locally on your changes still show the following errors:
```
python3 -m pycodestyle rock_paper_scissor.py
rock_paper_scissor.py:13:38: E203 whitespace before ':'
rock_paper_scissor.py:14:80: E501 line too long (90 > 79 characters)
rock_paper_scissor.py:27:80: E501 line too long (82 > 79 characters)
rock_paper_scissor.py:27:83: W291 trailing whitespace
rock_paper_scissor.py:28:80: E501 line too long (94 > 79 characters)
rock_paper_scissor.py:37:15: E225 missing whitespace around operator
rock_paper_scissor.py:37:30: E225 missing whitespace around operator
rock_paper_scissor.py:38:22: E225 missing whitespace around operator
rock_paper_scissor.py:40:17: E225 missing whitespace around operator
rock_paper_scissor.py:40:32: E225 missing whitespace around operator
rock_paper_scissor.py:41:22: E225 missing whitespace around operator
rock_paper_scissor.py:43:17: E225 missing whitespace around operator
rock_paper_scissor.py:43:32: E225 missing whitespace around operator
rock_paper_scissor.py:44:22: E225 missing whitespace around operator
rock_paper_scissor.py:46:17: E225 missing whitespace around operator
rock_paper_scissor.py:46:32: E225 missing whitespace around operator
rock_paper_scissor.py:47:22: E225 missing whitespace around operator
rock_paper_scissor.py:49:17: E225 missing whitespace around operator
rock_paper_scissor.py:49:32: E225 missing whitespace around operator
rock_paper_scissor.py:50:22: E225 missing whitespace around operator
rock_paper_scissor.py:52:17: E225 missing whitespace around operator
rock_paper_scissor.py:52:32: E225 missing whitespace around operator
rock_paper_scissor.py:53:22: E225 missing whitespace around operator
rock_paper_scissor.py:55:18: E225 missing whitespace around operator
rock_paper_scissor.py:55:33: E225 missing whitespace around operator
rock_paper_scissor.py:55:49: E225 missing whitespace around operator
rock_paper_scissor.py:55:64: E225 missing whitespace around operator
rock_paper_scissor.py:55:68: W291 trailing whitespace
rock_paper_scissor.py:56:16: E225 missing whitespace around operator
rock_paper_scissor.py:56:31: E225 missing whitespace around operator
rock_paper_scissor.py:74:8: E203 whitespace before ':'
rock_paper_scissor.py:79:80: E501 line too long (81 > 79 characters)
rock_paper_scissor.py:86:31: E228 missing whitespace around modulo operator
rock_paper_scissor.py:86:33: E225 missing whitespace around operator
rock_paper_scissor.py:87:80: E501 line too long (96 > 79 characters)
rock_paper_scissor.py:93:80: E501 line too long (90 > 79 characters)
rock_paper_scissor.py:99:46: E225 missing whitespace around operator
rock_paper_scissor.py:110:1: W293 blank line contains whitespace
rock_paper_scissor.py:110:1: W391 blank line at end of file
```

File: rock_paper_scissor.py (hunk 1 above)
Comment (SarthakRana) [reply rock_paper_scissor.py]: Hey @Pepper-Wood . Thanks for pointing this out. I'll get this checked by @mehta-raghav 👍

Comment (mehta-raghav) [reply rock_paper_scissor.py]: Hey, it seems my PEP8 changes didn't commit through. I've done it now. Please re-evaluate. Thanks and sorry for the inconvenience.
As for the 128 pound sign I went for a horizontal rule across the console. If its a bit much I'll revert it back.

@Pepper-Wood thanks for pointing this out. 😁 It'll help this pull request get merged.

**Review by SarthakRana — APPROVED** [review]
Looks good.

**Review by SarthakRana — APPROVED** [review]

**Pull request merged**